src/PsrContainerExtension.php

```diff
@@ -38,7 +41,11 @@ public function load(ContainerBuilder $container, array $config)
     {
         $container->setParameter('roave.behat.psr.container.included.file', $config['container']);
 
-        $loader = new YamlFileLoader($container, new FileLocator(__DIR__));
-        $loader->load('services.yml');
+        $definition = new Definition(ContainerInterface::class, ["%roave.behat.psr.container.included.file%"]);
```

Comment:
If I understand correctly, the `class` setting from the defunct `services.yml` has changed from `RoaveBehatPsrContainer` to `Psr\Container\ContainerInterface`... if that's the case, isn't it preferable to use a unique name here?